docs/changes/newsfragments/3688.n52xx

```diff
@@ -0,0 +1,4 @@
+- Update N52xx driver to fix warnings on initialization
```

Comment:
this file should be called 3688.improved_driver :) see here https://qcodes.github.io/Qcodes/community/contributing.html#pull-requests

Reply:
Thanks, noted and fixed :)